python/tvm/relay/op/contrib/arm_compute_lib.py

```diff
@@ -62,7 +72,7 @@ def partition_for_arm_compute_lib(mod, params=None):
             transform.InferType(),
             transform.MergeComposite(arm_compute_lib_pattern_table()),
             transform.AnnotateTarget("arm_compute_lib"),
-            transform.PartitionGraph(),
+            transform.PartitionGraph(optimize),
```

Comment:
Why cant we just run the optimize pass before partition graph here ? Is there a reason why it has to be inside PartitionGraph() ?

Reply:
This pass applies to each partitioned function, so it has to be called after the partitioned function has been created. See #6068

Reply:
Hmm, then why not after PartitionGraph ? (using kCompiler as a filter) ?

Reply:
As being said in that PR, we can definitely do that. In fact we are planning an RFC right now, so depending on one the RFC, we probably don't need this PR anymore.

Reply:
OK, sounds good :). I think PartitionGraph should just perform partitioning unless its really unavoidable to perform the needed post-processing after the pass.